flink-ml-benchmark/src/main/resources/naivebayes-benchmark.json

```diff
@@ -0,0 +1,44 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+{
+  "version": 1,
+  "NaiveBayes": {
+    "stage": {
+      "className": "org.apache.flink.ml.classification.naivebayes.NaiveBayes",
+      "paramMap": {
+        "smoothing": 1.0,
+        "featuresCol": "features",
```

Comment:
Do you think it would make user experience better by skipping parameters whose value is the same as the default value in the config file?